### ISSUE TITLE
Unable to recognize the types and methods defined in global.d.ts

### ISSUE THREAD

**Issue by ZhengXiaowei** [zed-industries/community#2201]
### Check for existing issues

- [X] Completed

### Describe the feature

Unable to recognize the types and methods defined in global.d.ts.

I have a `global.d.ts` file in the src directory, but when writing code in `*.ts` files, I cannot access the types and methods defined in `global.d.ts`.

e.g:
<img width="697" alt="image" src="https://github.com/zed-industries/community/assets/15809841/ddcf6529-0ffe-440c-9ff2-a0e1debb6023">
<img width="1226" alt="image" src="https://github.com/zed-industries/community/assets/15809841/6a8062e0-eeb8-4082-ab68-0e8798c06631">

**Comment by JosephTLyons** [comment]
Hi @ZhengXiaowei, this may be a naive thing to suggest (I'm not in the world of TypeScript often), but I added a `tsconfig.json` file and told it to look for everything `*.ts`-related, and it was able to find the global definitions.

**Comment by ZhengXiaowei** [comment]
> Hi @ZhengXiaowei, this may be a naive thing to suggest (I'm not really in the world of JavaScript / TypeScript), but I added a `tsconfig.json` file and told it to look for everything `*.ts`-related, and it was able to find the global definitions.
> 
> <img alt="SCR-20231103-jrhe" width="1608" src="https://user-images.githubusercontent.com/19867440/280311783-05685546-83c3-4157-8c5b-6349db906219.png">

I configured it to `tsconfig.json` but it's still not working.😭

**Comment by JosephTLyons** [comment]
I dont think you should have to, but did you try restarting the language server (command palette > `editor: restart language server`) to see if it picked up the changes? Can you include an image of the project tree?

**Comment by ZhengXiaowei** [comment]
I retry it today, and it's work for the typescript project, but it's doesn't work for the vue-typescript project.
it's my project tree:
<img width="399" alt="image" src="https://github.com/zed-industries/community/assets/15809841/d5a100f3-840c-48ee-a092-760533215fc8">

**Comment by ZhengXiaowei** [comment]
Hi @JosephTLyons, can this issue be fixed?
